packages/one-app-bundler/.eslintrc.json

```diff
@@ -0,0 +1,8 @@
+{
+  "extends": "amex",
+  "rules": {
+    "no-console": 0,
+    "import/extensions": 0, // ECMAscript modules explicitly require all extensions for relative imports
```

Comment:
Rather than disabling the rules it'd be more useful to adjust their options to enforce the ECMAScript Module requirements 🙏
Looks like there are a number of rules turned off, hopefully we can adjust them instead

Reply:
Yes, I was planning on bubbling both these rules up to the main config,

perhaps amex/cli and amex/esm, since these two rules specifically are getting more and more common to tweak.